python/hsfs/feature_view.py

```diff
@@ -2128,6 +2124,303 @@ def delete_all_training_datasets(self):
+
+        If no arguments is provided the method will return all feature monitoring configs
+        attached to the feature view, meaning all feature monitoring configs that are attach
+        to a feature in the feature view. If you wish to fetch a single config, provide the
+        its name. If you wish to fetch all configs attached to a particular feature, provide
+        the feature name.
+
+        !!! example
+            ```python3
+            # fetch your feature view
+            fv = fs.get_feature_view(name="my_feature_view", version=1)
+
+            # fetch all feature monitoring configs attached to the feature view
+            fm_configs = fv._get_feature_monitoring_configs()
+
+            # fetch a single feature monitoring config by name
+            fm_config = fv._get_feature_monitoring_configs(name="my_config")
+
+            # fetch all feature monitoring configs attached to a particular feature
+            fm_configs = fv._get_feature_monitoring_configs(feature_name="my_feature")
+
+            # fetch a single feature monitoring config with a particular id
+            fm_config = fv._get_feature_monitoring_configs(config_id=1)
+            ```
+
+        # Arguments
+            name: If provided fetch only the feature monitoring config with the given name.
+                Defaults to None.
+            feature_name: If provided, fetch only configs attached to a particular feature.
+                Defaults to None.
+            config_id: If provided, fetch only the feature monitoring config with the given id.
+                Defaults to None.
+
+        # Raises
+            `hsfs.client.exceptions.RestAPIError`.
+            `hsfs.client.exceptions.FeatureStoreException`.
+            ValueError: if both name and feature_name are provided.
+            TypeError: if name or feature_name are not string or None.
+
+        # Return
+            Union[`FeatureMonitoringConfig`, List[`FeatureMonitoringConfig`], None]
+                A list of feature monitoring configs. If name provided,
+                returns either a single config or None if not found.
+        """
+        # TODO: Should this filter out scheduled statistics only configs?
+        if not self._id:
+            raise FeatureStoreException(
+                "Only Feature Group registered with Hopsworks can fetch feature monitoring configurations."
+            )
+
+        fm_config_engine = (
+            feature_monitoring_config_engine.FeatureMonitoringConfigEngine(
+                feature_store_id=self._featurestore_id,
+                feature_view_id=self._id,
+                feature_view_name=self._name,
+                feature_view_version=self._version,
+            )
+        )
+
+        return fm_config_engine.get_feature_monitoring_configs(
+            name=name,
+            feature_name=feature_name,
+            config_id=config_id,
+        )
+
+    def _get_feature_monitoring_history(
```

Comment:
This is a duplicate of `feature_monitoring_config.get_history`. Instead users can do `fv.get_config.get_history`

Reply:
I realise that, I do feel like having to go through the config is less practical for the user because you have to get the config before being able to either:
- autocomplete with a method that does what you want
- look at the docs for the get_history function